src/main/kotlin/org/malachite/estella/process/domain/types.kt

```diff
@@ -0,0 +1,44 @@
+package org.malachite.estella.process.domain
+
+import org.malachite.estella.commons.models.offers.RecruitmentProcess
+import org.malachite.estella.commons.models.offers.RecruitmentStage
+import org.malachite.estella.commons.models.quizes.Quiz
+import org.malachite.estella.commons.models.tasks.Task
+import org.malachite.estella.offer.domain.OfferResponse
+import org.malachite.estella.offer.domain.toOfferResponse
+import java.sql.Date
+import java.sql.Timestamp
+
+data class RecruitmentProcessDto(
```

Comment:
Czysto teoretycznie z tego co pamiętam to DTO trzymamy w plikach typu ApplicationPayloads, więc pytanie o konwencje

Reply:
Ogólnie plik types powstał dlatego, że mamy multum konwencji, to wprowadziłem jeszcze jedną, typescriptową - to jakby komuś się przydała, to będzie precedens XD Ale ogólnie mogę coś pomyśleć, tylko nie wiem, czy tutaj to robić, czy jednak lepiej nie